### PR TITLE
chore(docs): add PR template. update links to the contributing doco

### PR DESCRIPTION
**Pull request by philipwindeyer**
## Description

- Minor update to the README
- Add PULL_REQUEST_TEMPLATE with a base check-list, including description of squash & merge message

## Related issues

Closes;
- https://github.com/hooroo/roo-ui/issues/267
- https://github.com/hooroo/roo-ui/issues/268

💪

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [x] You've requested a review from at least ONE person. Preferably a roo-ui maintainer (@angusfretwell, @talbet, @philipwindeyer)
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor verison bump to resolve security vulnerability in set-value`

### CODE REVIEW

**Comment by angusfretwell** [comment]
> You opt to "Squash and merge"
This is not necessary as it's the one option

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+---
+
+**PR check-list**
+
+🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀
+
+- [ ] Linked this to an [open issue](https://github.com/hooroo/roo-ui/issues) for context
+- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer (@angusfretwell, @talbet, @philipwindeyer)
+- [ ] You opt to "Squash and merge"
```
Comment (angusfretwell) [inline .github/PULL_REQUEST_TEMPLATE.md]: This is not necessary as it's the one option

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# A Pull Request! Nice one 🤙
```
Comment (angusfretwell) [inline .github/PULL_REQUEST_TEMPLATE.md]: Sorry to be fun police but could we remove this? Doesn't really add anything

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Description
+
+<!-- Add a short description here please] -->
```
Comment (angusfretwell) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe "Add a short description of your changes"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+🤨
+
+---
+
+**PR check-list**
+
+🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀
+
+- [ ] Linked this to an [open issue](https://github.com/hooroo/roo-ui/issues) for context
```
Comment (angusfretwell) [inline .github/PULL_REQUEST_TEMPLATE.md]: Hmm, I wonder if we really want to make this seem necessary? I think it's enough that it's a heading above

**Comment by angusfretwell** [comment]
Nice 💥

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (philipwindeyer) [reply .github/PULL_REQUEST_TEMPLATE.md]: 👍

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (philipwindeyer) [reply .github/PULL_REQUEST_TEMPLATE.md]: Booooo. k den

**Pull request merged**